### PR TITLE
Mab stra

### PR DESCRIPTION
**Pull request by mablue**
Thank you for sending your pull request.

## Summary
Explain in one sentence the goal of this PR / Strategy

Solve the issue: #___

## Quick strategy idea

- <change log #1>
- <change log #2>

### CODE REVIEW

File: user_data/strategies/mabStra.py [hunk 1]
```diff
@@ -9,58 +13,54 @@
...
+    buy_div_max = DecimalParameter(0, 2, decimals=4, default=2.25446, space='buy')
+    buy_div_min = DecimalParameter(0, 2, decimals=4, default=0.29497, space='buy')
+    # sell params
+    sell_mojo_ma_timeframe = IntParameter(2, 100, default=7, space='sell')
+    sell_fast_ma_timeframe = IntParameter(2, 100, default=14, space='sell')
+    sell_slow_ma_timeframe = IntParameter(2, 100, default=28, space='sell')
+    sell_div_max = DecimalParameter(0, 2, decimals=4, default=1.54593, space='sell')
+    sell_div_min = DecimalParameter(0, 2, decimals=4, default=2.81436, space='sell')
+
+    stoploss = -1
```
Comment (xmatthias) [inline user_data/strategies/mabStra.py:31]: I find this pretty dangerous, and would not want us to advertise running without stoploss - as this can result in a loss of 100% of the money in the account.
maybe change it to something reasonable (10-20% maybe?) ... ?

**Review by xmatthias — COMMENTED** [review]
Looking at the commit history - this are incremental PR's - each containing all files from the previous PR's.

Ideally (and that's what i meant in #164 ) - you'd create a new branch per PR - and then add one strategy (or strategy/hyperopt combo) to that.
Before creating the next branch, you switch back to the master branch - so your base is master again.
Otherwise, the "last" PR will contain all files from the previous pull requests - which (while in theory it's not a problem) can lead to merge conflicts.

File: user_data/strategies/mabStra.py (hunk 1 above)
Comment (mablue) [reply user_data/strategies/mabStra.py:31]: Yes why not??!

**Review by xmatthias — APPROVED** [review]
Great!
Thanks for sharing your work! :+1:
